h/static/scripts/test/root-thread-test.js

```diff
@@ -212,6 +212,19 @@ describe('rootThread', function () {
       assert.isTrue(threadFilterFn(fakeBuildThread));
     });
 
+    it('generates a thread filter function to match orphans', function () {
+      fakeBuildThread.reset();
+      fakeBuildThread.annotation = {$orphan: true};
+
+      fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
+        {selectedTab: 'orphan'});
+
+      rootThread.thread(fakeAnnotationUI.state);
+      var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
+
+      assert.isTrue(threadFilterFn(fakeBuildThread));
```

Comment:
I missed this before but this test is broken. The thread filter function takes a _Thread_ as an argument, not the `buildThread` function, which is what _fakeBuildThread_ returns.

This kind of thing is why I would really like to introduce a type checker in our code :)